src/docs/arch-eviction.dox

```diff
@@ -86,29 +86,8 @@ cache contains at least this many bytes of updates.
 It is worth noting that for all eviction target and trigger sizes, the target size must be lower
 than its trigger size counterpart.
 
-@section eviction_locking Locking
-
-Internally eviction uses the following locks:
-
-- \c cache->evict_pass_lock - Guarantees that only one eviction server is running at most.
-- \c cache->evict_walk_lock - Locked when eviction walks a tree.
-- \c cache->evict_queue_lock - Access to cache->evict_queues.
-- \c queue->evict_lock - Access to a specific eviction queue.
```

Comment:
I intentionally removed descriptions of locks since they are just copied from the source. The purpose of the page is to document locking hierarchy which is sometimes hard to get. Locks descriptions are easy to find where they are declared and I didn't want to overload the page with unnecessary information.

If you believe that this kind of table would be useful, I can make a script that would automatically generate one from sources. This method will also guarantee that the table is always up to date.

Reply:
Okay, that's fine.  No need to do more.